OpenRA.Mods.Common/Traits/ActorSoldNotifier.cs

```diff
@@ -0,0 +1,41 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2021 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Plays an audio notification when the actor is sold.")]
+	public class ActorSoldNotifierInfo : TraitInfo
+	{
+		[NotificationReference("Speech")]
+		[Desc("The audio notification type to play.")]
+		public string Notification = "UnitSold";
```

Comment:
Why define this as `UnitSold` when all our uses are for `StructureSold`?

Reply:
I wanted to set that up first, then I noticed we don't have https://github.com/OpenRA/OpenRA/issues/3798 yet.